ssh/tailssh/incubator.go

```diff
@@ -283,7 +283,20 @@ func beIncubator(args []string) error {
 			Foreground: true,
 		}
 	}
-	return cmd.Run()
+	err = cmd.Run()
+	if ee, ok := err.(*exec.ExitError); ok {
+		ps := ee.ProcessState
+		code := ps.ExitCode()
+		if code < 0 {
+			// TODO(bradfitz): do we need to also check the syscall.WaitStatus
+			// and make our process look like it also died by signal/same signal
+			// as our child process? For now we just do the exit code.
+			fmt.Fprintf(os.Stderr, "[tailscale-ssh: process died: %v]\n", ps.String())
```

Comment:
Should this be a more formal log line with `logf`? (I can't quite tell if you have access to that here...)

Reply:
Looks like `logf := logger.Discard` so I guess not.

Reply:
logf at this point is a weirdo thing that writes via a high fd up to the child process, I believe.

This is definitely a weird environment.